snap/snapcraft.yaml

```diff
@@ -0,0 +1,63 @@
+name: tempest
+adopt-info: tempest
+summary: OpenStack Integration Test Suite
+description: |
+ Tempest is a set of integration tests to be run against a live
+ OpenStack cluster. Tempest has batteries of tests for OpenStack
+ API validation, Scenarios, and other specific tests useful in
+ validating an OpenStack deployment.
+confinement: strict
+grade: stable
+base: core22
+
+apps:
+  tempest:
+    command: bin/tempest
+    completer: usr/share/bash-completion/completions/tempest
+    plugs:
+      - network-observe
+      - network
+      - home
+parts:
+  tempest:
+    plugin: python
+    source: .
```

Comment:
During my work on python-tempestconf, I realized that here we should use source for tempest project (example https://tarballs.opendev.org/openstack/tempest/tempest-master.tar.gz), instead of sourcing `.`. Since this is snap repo and all files provided here are related for snap and not for tempest. What do you say?

Reply:
That master tarball is quite outdated. Anyway, we can pull source from VCS but I don't think building the master branch is a good idea. I think we should stick to released versions, i.e. checkout specific tags. So more or less it is the same thing with using requirements and specifying the package version there.

Only caveat would be, once Yoga switches to EM, we could create a `stable/yoga` branch and in requirements.txt, switch from package names to
```
git+https://opendev.org/openstack/tempest.git@yoga-last
git+https://opendev.org/openstack/cinder-tempest-plugin.git@yoga-last
...
```
since the `*-last` tags will be available. See example in [ussuri release page](https://releases.openstack.org/ussuri/#tempest-plugins)